drivers/display/ssd16xx.c

```diff
@@ -974,6 +975,44 @@ static int ssd16xx_init(const struct device *dev)
 	return ssd16xx_controller_init(dev);
 }
 
+#ifdef CONFIG_PM_DEVICE
+static int ssd16xx_pm_action(const struct device *dev,
+			     enum pm_device_action action)
+{
+	int err = 0;
+	const struct ssd16xx_config *config = dev->config;
+
+	switch (action) {
+	case PM_DEVICE_ACTION_RESUME:
+		err = ssd16xx_controller_init(dev);
+		if (err < 0) {
+			return err;
+		}
+		break;
+
+	case PM_DEVICE_ACTION_SUSPEND:
+		ssd16xx_busy_wait(dev);
+		err = ssd16xx_write_uint8(dev, SSD16XX_CMD_SLEEP_MODE, 1);
```

Comment:
You should probably use the SSD16XX_SLEEP_MODE_DSM constant here. Some controllers have a couple of different sleep modes